interoperability_report.py

```diff
@@ -60,7 +62,7 @@ def subscriber(
 
         If the Shape Application achieves one step, it will print an specific
         string pattern. The function will recognize that pattern and it will
-        continue also to the next step, waiting again for the next
+        continue to the next step too, waiting again for the next
         corresponding pattern to be recognized. If the Shape Application
         stops at some step, the function will not recognized the expected pattern
         (or it will recognized an error pattern), it will save
```

Comment:
```suggestion
        and waits for the next input string from the ShapeApplication. If the
        ShapeApplication stops at some step, it prints an error message. When this
        function matches an error string (or doesn't match an expected pattern
        in the specified timeout), the corresponding ReturnCode is saved in
        produced_code[produced_code_index] and the process finishes.
```

Reply:
delete also the next line (I cannot add a comment from GitHub).

Please, review also the documentation of other similar functions